### PR TITLE
Add new methods to read and write Emissivity factor to the sensor.

### PR DESCRIPTION
**Pull request by newts**
- **Describe the scope of your change--i.e. what the change does and what parts
  of the code were modified.**  This will help us understand any risks of integrating
  the code.

No changes, added new methods to read and write Emissivity factor which was needed to help get better accuracy for making fever thermometer as well as adding missing feature of the library to take advantage of the sensor.

Shouldn't break anything.  testing more now for casual use.  no formal testing but operationally seems to work.

This PR also includes  required write16 and crc8 utility functions for internal use to make it easier to add other features if needed.

### CODE REVIEW

File: Adafruit_MLX90614.h [hunk 1]
```diff
@@ -53,11 +53,17 @@ class Adafruit_MLX90614 {
   double readAmbientTempC(void);
   double readObjectTempF(void);
   double readAmbientTempF(void);
+  uint16_t readEmmissivityReg(void);
```
Comment (ladyada) [inline Adafruit_MLX90614.h:56]: please rename functions to `Emissivity` (one m)

Comment (newts) [reply Adafruit_MLX90614.h:56]: Foo!   i kept getting that wrong and also emissitivity...  will fix.  thanks for catching.

Comment (newts) [reply Adafruit_MLX90614.h]: Is fixed now.  I didn't update the version number in the library file, I figure that was up to you.

do you need a new PR?

**Comment by ladyada** [comment]
@siddacious plz add to actionslist :)

File: Adafruit_MLX90614.h (hunk 1 above)
Comment (ladyada) [reply Adafruit_MLX90614.h]: nope!

**Pull request merged**